src/common/pico_binary_info/include/pico/binary_info/code.h

```diff
@@ -131,7 +131,7 @@ static const struct _binary_info_named_group __bi_lineno_var_name = { \
 #define bi_pin_range_with_func(plo, phi, func)       __bi_encoded_pins_with_func(BI_PINS_ENCODING_RANGE | ((func << 3)) | ((plo) << 7) | ((phi) << 12))
 
 #define bi_pin_mask_with_name(pmask, label)          __bi_pins_with_name((pmask), (label))
-// names are sperated by | ... i.e. "name1|name2|name3"
+// names are seperated by | ... i.e. "name1|name2|name3"
```

Comment:
The correct spelling is "separated" :wink:

Reply:
I am so embarrassed😨 
I fixed. Thank you for reviewing!